spec/index.bs

```diff
@@ -1651,7 +1651,7 @@ Queues {#queues}
 
 <script type=idl>
 interface GPUQueue {
-    void submit(sequence<GPUCommandBuffer> buffers);
+    void submit(GPUCommandBuffer commandBuffer);
```

Comment:
This isn't really a hot path as it should only happen a couple times per frame. Also calling `GPUQueue.submit` will probably force submitting at the native graphics API level which could be expensive. If we do this we should consider having "shallow flushes".  Basically my point is that this is more complicated than it looks.

Reply:
Good point.

Reply:
FYI, vkQueueSubmit([A,B]); is very different from vkQueueSubmit(A); vkQueueSubmit(B); in terms of how the driver is allowed to reorder them. Not sure about other APIs.

Reply:
Could you point to the concrete place in the Vulkan spec where this is described?